webapp/src/modules/transfer/reducer.js

```diff
@@ -0,0 +1,48 @@
+import {
+  TRANSFER_PARCEL_REQUEST,
+  TRANSFER_PARCEL_SUCCESS,
+  TRANSFER_PARCEL_FAILURE,
+  CLEAN_TRANSFER
+} from './actions'
+
+const INITIAL_STATE = {
+  data: {},
+  loading: false,
+  error: null
+}
```

Comment:
_Reviewer_: I have a few doubts regarding the `transfer` domain.

The two main things are:
- It stands as a top level domain. My first instinct was to place it under parcels, but that also seems weird to me. I'm not sure where it's best (but it's easy enough to move around). What do you think?
- The structure of the data is:
```javascript
{
  hash: string,
  oldOwner: string,
  newOwner: string
}
```
which has the problem of having to `clean`  the transfer ( [see here](https://github.com/decentraland/marketplace/pull/34/files#diff-3157d100f0877a789f687b803efc6ffeR67) ) after closing the modal that creates it. Another option may be to have an object `{ 'x,y': { current data } }` as we use with parcels, but it seemed unnecessarily complex. Let me know if you have a better option